report.md

```diff
@@ -462,13 +462,7 @@ In total five unit-tests were added to increase the branch coverage of the `Bina
 
 ## Self-assessment: Way of working
 
-Current state according to the Essence standard: ...
-
-Was the self-assessment unanimous? Any doubts about certain items?
-
-How have you improved so far?
-
-Where is potential for improvement?
+We would say that we are in the state *In place*. We all use the same practices and tools to conduct our work. Furthermore, all team members have access to the same practices and tools that we use. There are some inconsistencies with how we do some parts of the work. For example, when it comes to reviewing pull requests some of our group members adapted a new practice for writing comments. They begin each comment with SD (should do), CD (could do), or MD (must do) to indicate how important the comment is. This is probably a good practice and suits our context, but all group members are not adapting to this way-of-working. Consequently, we do not fulfill the first criteria from the *Working well* state which is “Team members are making progress as planned by using and adapting the way-of-working to suit their current context.” To reach this state we should work on adapting our way-of-working to better suit our context, and ensure that everyone is on board with new practices such as writing SD, CD, or MD at the beginning of each pull request comment.
```

Comment:
CD: i could stop so we work the same way :)

Reply:
No we need something to write about, so it is good.